core/client/app/templates/components/gh-profile-image.hbs

```diff
@@ -1,11 +1,11 @@
 <figure class="account-image js-file-upload">
     {{#unless hasUploadedImage}}
-        <div class="placeholder-img" style={{defaultImage}}></div>
-
         {{#if hasEmail}}
```

Comment:
Yes definitely undesirable. Two possible solutions:
1. Gravatar allows for a default image to be specified for failed lookups. As long as the default image url is accessible (not behind auth). 
2. Tie into image loading events to know when to show the default image vs the gravatar loaded one (assuming it loads)